examples/qt/README.md

```diff
@@ -0,0 +1,45 @@
+# libsigc++ with Qt
+
+It is possible to use libsigc++ with Qt.  However, because of the signals/slots
+mechanism of Qt, there is some setup that must be done in order for this to
+happen correctly.
+
+Steps to use libsigc++ with Qt:
+1. In your .pro file, add `CONFIG += no_keywords`.  This configures Qt to not
```

Comment:
This assumes that the person is using qmake, I think. But Qt doesn't require the use of qmake.

Reply:
Also, I think it's worth saying that this (using the Q_* macros) is generally good advice when writing Qt code.

Reply:
I'm generally in agreement with that, I'll go update this in a bit.  While I don't disagree that using the Q_* macros is a good idea, that's generally not what happens in most Qt code that I have seen(even Qt's own examples use `emit`/`signals`/`slots` and not the Q_* macros).